packages/tools/pie-icons-vue/test/IconAlcohol.test.js

```diff
@@ -29,4 +38,55 @@ describe('IconAlcohol', () => {
         expect(wrapper.classes()).toContain(`c-pieIcon--${iconSize}`);
         expect(wrapper.classes()).toHaveLength(3);
     });
+
+    test.each([
+        'xxs',
+        'xxxl',
+        '',
+    ])('should not allow invalid sizes - %s', (iconSize) => {
```

Comment:
I would add somewhere in your test description that these tests are for a regular icon (rather than for the `Large` variant)